modules/ossm-control-plane-configuration-migration-requirements.adoc

```diff
@@ -0,0 +1,67 @@
+// Module included in the following assemblies:
+
+// * service-mesh-docs-main/migrating/cluster-wide/ossm-migrating-cluster-wide-assembly.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="ossm-about-cluster-wide-migration_{context}"]
+= Control plane configuration migration requirements
+
+During the migration process, two cluster-wide control planes run in the same cluster while the data plane namespaces are gradually migrated to the {SMProductName} 3.0 installation. One control plane is associated with the {SMProductName} {SMv2Version} installation and the other is associated with the {SMProduct} 3.0 installation. You must carefully plan out the migration steps to avoid possible conflicts between the two control planes.
+
+[id="ossm-cluster-wide-migration-root-certificate_{context}"]
+== Root certificate
+
+During the migration, both control planes must share a root certificate. This can be achieved by installing the 3.0 control plane into the same namespace as 2.6 control plane. The migration procedures show how to verify that the root certificate is shared.
+
+[id="ossm-cluster-wide-migration-discovery-selectors_{context}"]
+== Discovery selectors and namespace access
+
+Both control planes must have access to all namespaces in the mesh. During the migration, some proxies are controlled by the 3.0 control plane while other proxies remain controlled by the 2.6 control plane. To ensure that mesh communication works during the migration, both control planes must be aware of the same set of services. Service discovery is provided by `istiod` component, which runs in the control plane namespace.
+
+In the {SMProduct} 3.0 installation, you can control how {istio} discovers services by using discovery selectors. When using discovery selectors, ensure that the `discoverySelectors` expression that is defined in the {SMProduct} 3.0 {istio} resource match the namespaces that comprise the {SMProduct} 2.6 mesh. You might have to add additional labels to the {SMProduct} 2.6 application namespaces to ensure that they are captured in the {SMProduct} 3.0 installation. For more information, see "Scoping the service mesh with DiscoverySelectors".
+
+[NOTE]
+====
+In {SMProduct} 2.6 installations, the `maistra.io/member-of` label is automatically created. This label cannot be used because it is automatically removed during the migration process.
+====
+
+[id="ossm-cluster-wide-migration-network-policies_{context}"]
+== Network policies
+
+By default, {SMProduct} 2.6 manages network policies that block traffic to the 3.0 control plane. 
+
+For both control planes, during the migration ensure that network policies do not block traffic:
```

Comment:
```suggestion
For both control planes, during the migration ensure that network policies do not block traffic between the following entities:
```
This might be subjective, but I stumbled a little on the original wording of this list. I wonder if it might be a clearer if the "between" phrasing is moved into the lead-in phrasing. This is just a suggestion.

Reply:
done